### PR TITLE
Update for office hour change to slack

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

**Comment by nedbat** [comment]
@jonkiparsky Does this look OK?

File: officehours.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 ---
 
-Boston Python holds a weekly Zoom office hour on Mondays at noon, sponsored by [edX](https://edx.org).
+Boston Python holds a weekly Zoom office hour on Mondays at noon.
 
 This is an unstructured time to drop in to ask questions or just chat. There is no set topic, anything goes: Python, Boston Python, specialized technical topics, adjacent technologies, it's all good.  Any expertise level is fine! For now, this is an experiment. We may change the structure, pace, or timing as we see what happens.
 
-The office hour happens on Zoom. The [event page on Meetup](https://www.meetup.com/bostonpython/events) has the link.  You do not have to sign up ahead of time, **just drop in**.  A running agenda for the Boston Python office hour is a Google doc [available here](https://docs.google.com/document/d/1LKSWk5SEZrbz7e93GB_ePMxKKEJ-QnUofmYTT9uSdUA). See you there!
+The office hour happens on Zoom. The [event page on Meetup](https://www.meetup.com/bostonpython/events) has the link.  You do not have to sign up ahead of time, **just drop in**.  Questions and topics can be addde in the [#office-hour][office-hour-channel] channel in our [Slack workspace][slack]. See you there!
```
Comment (bhrutledge) [inline officehours.md:11]:
```suggestion
The office hour happens on Zoom. The [event page on Meetup](https://www.meetup.com/bostonpython/events) has the link.  You do not have to sign up ahead of time, **just drop in**.  Questions and topics can be added in the [#office-hour][office-hour-channel] channel in our [Slack workspace][slack]. See you there!
```

File: events.md [hunk 2]
```diff
@@ -4,10 +4,11 @@ sidebar_link: true
...
 ---
 
-Boston Python runs three events most months:
+Boston Python runs four kinds of events:
 
 - **Presentation nights** feature presentations by members.  These are technical peer-to-peer presentations.  Anyone can do one!  Sometimes we do [lightning talks](lightning.md).
-- **Project nights** are unstructured times to sit with people with similar interests, and do whatever you would like to do.
+- **Collaboration nights** are unstructured times to sit with people with similar interests, and do whatever you would like to do.
 - **Python Study Group nights** feature presentations by members who are beginner-to-intermediate. Each presenter chooses a topic that they're unfamiliar with, researches it, and then teaches the group what they've learned.
+- **Office hour** is a weekly hour to drop in and ask anything on your mind.
```
Comment (bhrutledge) [inline events.md:12]:
```suggestion
- [**Office hour**](officehour.md] is a weekly hour to drop in and ask anything on your mind.
```

File: officehours.md [hunk 3]
```diff
@@ -1,11 +1,14 @@
 ---
-title: Office Hours
+title: Office Hour
```
Comment (bhrutledge) [inline officehours.md:2]: How about renaming this file to `officehour.md`, and adding:

```suggestion
title: Office Hour
redirect_from:
    - /officehours
```

**Comment by nedbat** [comment]
More comment welcome as always.

**Comment by jonkiparsky** [comment]
Looks good to me, Ned. Thanks for updating.

**Pull request merged**